charts/rancher-logging/0.2.5/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
Is this a backport, or just the chart version change ? Needs some issue linked which explains what/why its changing.